projects/demo/src/modules/components/island/examples/3/index.less

```diff
@@ -39,15 +39,18 @@
     }
 
     [tuiaction] {
+        cursor: default;
         box-shadow: none;
         border: 1px solid var(--tui-base-03);
         border-bottom: none;
+        will-change: tranform;
```

Comment:
is it necessary? 

`Warning: will-change is intended to be used as a last resort, in order to try to deal with existing performance problems. It should not be used to anticipate performance problems.` (mdn)

Reply:
fixed